applications/registration-react/Dockerfile

```diff
@@ -1,18 +1,14 @@
 FROM node:12-alpine
 
 #the only reason why we need git here, is that we have designsystem in another github repo
-RUN apk add --no-cache git
+RUN apk add --no-cache git gettext
```

Comment:
Where is gettext used?

Reply:
> Where is gettext used?

`envsubst` command inside `applications/registration-react/entrypoint.sh`